spring-web/src/main/java/org/springframework/http/HttpStatusCode.java

```diff
@@ -88,6 +88,15 @@ default boolean isSameCodeAs(HttpStatusCode other) {
 		return value() == other.value();
 	}
 
+	/**
+	 * Checks whether the given status code is a well-known HTTP status code or not
+	 * @param statusCode the HTTP status code (potentially non-standard)
+	 * @return {@code true} if the status code corresponds to a standard HTTP status code
+	 */
+	static boolean isWellKnownStatusCode(HttpStatusCode statusCode) {
+		return HttpStatus.resolve(statusCode.value()) != null;
+	}
```

Comment:
How about this? 
```java
	boolean isWellKnown() {
		return HttpStatus.resolve(this.value()) != null;
	}
```

Reply:
Very good point, thank you. I changed it to a default method on the interface.